app/src/config/smmtConfigurationLoader.js

```diff
@@ -9,6 +9,8 @@ let logger;
 let smmtApiKey;
 
 function getSmmtApiKey() {
+  logger.debug({ context: { ciphertext: env.SMMT_API_KEY } }, 'Attempting to decrypt ciphertext.');
```

Comment:
I would at least partially obfuscate the encrypted `SMMT_API_KEY` Since we do not have full control of where and how long the logs will be retained. 

Without access to relevant KMS key it doesn't give much, but still it decreases a probability of someone gaining access to the api key.

Not a huge priority but it doesn't cost us much, and the encrypted key in the logs doesn't give that much value (Perhaps snip out the middle, worst that could happen is that someone mistakingly added or deleted characters at the beginning or the end)

Reply:
but it is generated only when debug log level is set
Is it still an issue?

Reply:
As long as debug is not set on pp/prd it won't be.